cdc/api/v2/api.go

```diff
@@ -14,25 +14,62 @@
 package v2
 
 import (
+	"github.com/pingcap/tiflow/cdc/model"
+	"github.com/pingcap/tiflow/pkg/upstream"
+	"net/http"
+
 	"github.com/gin-gonic/gin"
 	"github.com/pingcap/tiflow/cdc/api/middleware"
 	"github.com/pingcap/tiflow/cdc/capture"
+	"github.com/pingcap/tiflow/cdc/owner"
 )
 
 // OpenAPIV2 provides CDC v2 APIs
-type OpenAPIV2 struct {
+type openAPIV2 struct {
```

Comment:
Why is this struct private?

Reply:
hmm, it seems that being public makes more sense. I edited it private since it's private in API v1.